minimesos/src/main/java/com/containersol/minimesos/cluster/ClusterRepository.java

```diff
@@ -22,6 +22,7 @@
     /**
      * Loads representation of the running cluster
      *
+     * @param factory TODO(mwl)
```

Comment:
I think it's better to just remove the @param from here. It's pretty self documenting anyway.